test/format/failure_configs/CONFIG.WT-11064

```diff
@@ -0,0 +1,229 @@
+#  TABLE PARAMETERS: table 1
+############################################
+table1.btree.compression=zstd
+table1.btree.dictionary=0
+table1.btree.internal_key_truncation=1
+table1.btree.internal_page_max=17
+table1.btree.key_max=52
+table1.btree.key_min=20
+table1.btree.leaf_page_max=13
+table1.btree.memory_page_max=7
+table1.btree.prefix_len=0
+table1.btree.prefix_compression=1
+table1.btree.prefix_compression_min=3
+table1.btree.reverse=0
+table1.btree.split_pct=97
+table1.btree.value_max=1389
+table1.btree.value_min=7
+table1.disk.checksum=on
+table1.disk.firstfit=1
+table1.ops.pareto=0
+table1.ops.pareto.skew=91
+table1.ops.pct.delete=0
+table1.ops.pct.insert=35
+table1.ops.pct.modify=0
+table1.ops.pct.read=65
+table1.ops.pct.write=0
+table1.ops.truncate=0
+table1.runs.mirror=0
+table1.runs.source=table
+table1.runs.type=row-store
+############################################
+#  TABLE PARAMETERS: table 2
+############################################
+table2.btree.bitcnt=4
+table2.btree.compression=none
+table2.btree.internal_key_truncation=1
+table2.btree.internal_page_max=9
+table2.btree.leaf_page_max=13
+table2.btree.memory_page_max=10
+table2.btree.split_pct=76
+table2.disk.checksum=unencrypted
+table2.disk.firstfit=0
+table2.ops.pareto=1
+table2.ops.pareto.skew=74
+table2.ops.pct.delete=36
+table2.ops.pct.insert=48
+table2.ops.pct.modify=16
+table2.ops.pct.read=0
+table2.ops.pct.write=0
+table2.ops.truncate=0
+table2.runs.mirror=0
+table2.runs.source=table
+table2.runs.type=row-store
+############################################
+#  TABLE PARAMETERS: table 3
+############################################
+table3.btree.bitcnt=8
+table3.btree.compression=zstd
+table3.btree.internal_key_truncation=1
+table3.btree.internal_page_max=17
+table3.btree.leaf_page_max=9
+table3.btree.memory_page_max=9
+table3.btree.split_pct=74
+table3.disk.checksum=off
+table3.disk.firstfit=0
+table3.ops.pareto=0
+table3.ops.pareto.skew=28
+table3.ops.pct.delete=14
+table3.ops.pct.insert=14
+table3.ops.pct.modify=2
+table3.ops.pct.read=68
+table3.ops.pct.write=2
+table3.ops.truncate=0
+table3.runs.mirror=0
+table3.runs.source=file
+table3.runs.type=row-store
+############################################
+#  TABLE PARAMETERS: table 4
+############################################
+table4.btree.compression=none
+table4.btree.dictionary=0
+table4.btree.internal_key_truncation=1
+table4.btree.internal_page_max=12
+table4.btree.leaf_page_max=11
+table4.btree.memory_page_max=8
+table4.btree.repeat_data_pct=39
+table4.btree.split_pct=57
+table4.btree.value_max=1453
+table4.btree.value_min=19
+table4.disk.checksum=uncompressed
+table4.disk.firstfit=0
+table4.ops.pareto=0
+table4.ops.pareto.skew=47
+table4.ops.pct.delete=11
+table4.ops.pct.insert=12
+table4.ops.pct.modify=74
+table4.ops.pct.read=0
+table4.ops.pct.write=3
+table4.ops.truncate=0
+table4.runs.mirror=0
+table4.runs.source=file
+table4.runs.type=variable-length column-store
```

Comment:
Change it to row-store as VLCS is not fully supported in 5.0 and older.

Reply:
done! [7a7da52](https://github.com/wiredtiger/wiredtiger/pull/9690/commits/7a7da52b5c5b83396135abcb3fb9bca17cd91dc8)